graql/grammar/Graql.g4

```diff
@@ -111,20 +111,17 @@ pattern             :   pattern_statement
                     |   pattern_disjunction
                     |   pattern_negation
                     ;
+
 pattern_conjunction :   '{' patterns '}' ';' ;
 pattern_disjunction :   '{' patterns '}'  ( OR '{' patterns '}' )+  ';' ;
-
-pattern_negation    :   NOT '{' patterns '}' ';' ;
+pattern_negation    :   NOT '{' pattern '}' ';' ;
```

Comment:
As discussed, `pattern_negation` should be `NOT { patterns }`, taking in one or more patterns. If there is more than one pattern provided, it will be a negation of a conjunction as it should.